packages/pds/tests/proxied/__snapshots__/thread.test.ts.snap

```diff
@@ -0,0 +1,812 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Just curious—are the proxied snapshots pretty much identical to the non-proxied versions?

Reply:
pretty much. only 3 differences:
- there are fewer of them
- labels are not in them (yet)
- the img links changed

I was fairly rigorous in parsing through them to verify. but we may want to do so again once we have labels in the appview